pkgs/applications/networking/browsers/chromium/upstream-info.json

```diff
@@ -2,16 +2,22 @@
   "stable": {
     "version": "86.0.4240.111",
     "sha256": "05y7lwr89awkhvgmwkx3br9j4ap2aypg2wsc0nz8mi7kxc1dnyzj",
-    "sha256bin64": "10aqiiydw4i3jxnw8xxdgkgcqbfqc67n1fbrg40y54kg0v5dz8l6"
+    "sha256bin64": "10aqiiydw4i3jxnw8xxdgkgcqbfqc67n1fbrg40y54kg0v5dz8l6",
+    "profile": "chrome-linux-master-1597082402-c5bd486642405d828e9e780a3fe4cf0c960a0b58.profdata",
+    "profile-sha256": "0ifxvii6fzx5aq1wlksz49qrljvv58779y472mlf5y0vxjk5knkd"
```

Comment:
Would be better to use `pgo-profile{,-sha256}` here (or scope them under a `pgo` key) to keep thinks clearer / easier to read and understand (though I could also change that later).